api/cases/case-list.spec.js

```diff
@@ -112,6 +112,98 @@ describe('case-list spec', () => {
         });
     });
 
+
+    describe('when 2 cases exist, one with valid and other with missing case reference', () => {
+        const createdDate = new Date();
+        const updatedDate = new Date();
+
+        beforeEach(() => {
+            caseData.length = 0;
+            caseData.push({
+                id: 987654321,
+                case_data: {
+                    caseReference: '123-123-123',
+                    appeal: {
+                        appellant: {
+                            name: {
+                                firstName: 'Louis',
+                                lastName: 'Houghton'
+                            }
+                        },
+                        benefitType: {
+                            code: 'PIP'
+                        }
+                    }
+                },
+                created_date: createdDate,
+                last_modified: updatedDate
+
+            });
+            caseData.push({
+                id: 987654322,
```

Comment:
Is this the same response that the template function produces? Just checking to make sure the output of that would omit the field rather than add `case_reference: undefined` for example

Reply:
Yes, its the same response after the template function produces the json, it omits the case_reference instead of adding it with `undefined`